crates/services/gas_price_service/src/fuel_gas_price_updater.rs

```diff
@@ -14,48 +16,55 @@ pub use fuel_gas_price_algorithm::{
         RecordedBlock,
     },
 };
-use std::num::NonZeroU64;
 
 #[cfg(test)]
 mod tests;
 
 pub mod fuel_core_storage_adapter;
+pub mod fuel_da_source_adapter;
 
-pub struct FuelGasPriceUpdater<L2, Metadata> {
+pub struct FuelGasPriceUpdater<L2, Metadata, DaSource> {
     inner: AlgorithmUpdater,
     l2_block_source: L2,
     metadata_storage: Metadata,
+    #[allow(dead_code)]
+    da_source: DaSource,
```

Comment:
Is this still unused?

Reply:
it will be until we plug in the function call, ref: https://github.com/FuelLabs/fuel-core/pull/2155#discussion_r1741636419